src/Umbraco.Core/Persistence/Dtos/AxisDefintionDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using NPoco;
+
+namespace Umbraco.Core.Persistence.Dtos
+{
+    public class AxisDefintionDto
```

Comment:
This DTO is still public, can we also change it to internal?

Reply:
Done plus `UserNotificationDto` as well.